VirtoCommerce.QuoteModule.Data/Observers/LogQuoteChangesObserver.cs

```diff
@@ -20,47 +20,47 @@ public LogQuoteChangesObserver(IChangeLogService changeLogService)
         #region IObserver<QuoteRequestChangeEvent> Members
 
         public void OnCompleted()
-		{
-		}
+        {
+        }
+
+        public void OnError(Exception error)
+        {
+        }
+
+        public void OnNext(QuoteRequestChangeEvent value)
+        {
```

Comment:
Why this code is commented? Probably need to leave it to write to the changelog.